src/main/java/com/dimsirka/animalservice/controllers/ContactController.java

```diff
@@ -0,0 +1,16 @@
+package com.dimsirka.animalservice.controllers;
+
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+
+@Controller
+public class ContactController {
+
+    @GetMapping({"/contact"})
+    public String getAll() {
+        return "contact/contactUs";
+    }
+
+
```

Comment:
Move it to HomeController

Reply:
![image](https://user-images.githubusercontent.com/33618575/118694205-7ebb6880-b814-11eb-9436-4f30eb412102.png)
Fixed